tests/integration/mininet_tests.py

```diff
@@ -155,14 +155,7 @@ def verify_events_log(self, event_log, timeout=10):
 
     def test_untagged(self):
         """All hosts on the same untagged VLAN should have connectivity."""
-        event_log = os.path.join(self.tmpdir, 'event.log')
-        controller = self._get_controller()
-        sock = self.env['faucet']['FAUCET_EVENT_SOCK']
-        # Relying on a timeout seems a bit brittle;
-        # as an alternative we might possibly use something like
-        # `with popen(cmd...) as proc` to clean up on exceptions
-        controller.cmd(mininet_test_util.timeout_cmd(
-            'nc -U %s > %s &' % (sock, event_log), self.EVENT_LOGGER_TIMEOUT))
+        self._set_event_log(self.EVENT_LOGGER_TIMEOUT)
```

Comment:
the name implies that it's setting the event log to a timeout. Maybe _enable_event_log?